cpp_utils/tests/BLETests/Arduino/BLE_notify/BLE_notify.ino

```diff
@@ -24,7 +24,6 @@
 #include <BLE2902.h>
 
 BLEServer *pServer = NULL;
-BLECharacteristic *pCharacteristic;
```

Comment:
This broke code.  If we look within the loop, we still find references to pCharacteristic.  Moving this to be private within setup wouldn't have compiled.